dashboard/src/components/Config/AppRepoList/AppRepoList.tsx

```diff
@@ -155,80 +184,83 @@ function AppRepoList() {
           )
         }
       />
-      {!supportedCluster ? (
-        <Alert theme="warning">
-          <h5>Package Repositories can't be managed from this cluster.</h5>
-          <p>
-            Currently, the Package Repositories must be managed from the default cluster (the one on
-            which Kubeapps has been installed).
-          </p>
-          <p>
-            Any <i>global</i> Package Repository defined in the default cluster can be later used
-            across any target cluster.
-            <br />
-            However, <i>namespaced</i> Package Repositories can only be used on the default cluster.
-          </p>
-        </Alert>
-      ) : (
-        <div className="page-content">
-          {errors.fetch && (
-            <Alert theme="danger">
-              An error occurred while fetching repositories: {errors.fetch.message}
-            </Alert>
-          )}
-          {errors.delete && (
-            <Alert theme="danger">
-              An error occurred while deleting the repository: {errors.delete.message}
-            </Alert>
-          )}
-          {!errors.fetch && (
-            <>
-              <LoadingWrapper
-                className="margin-t-xxl"
-                loadingText="Fetching Package Repositories..."
-                loaded={!isFetchingElem.repositories}
-              >
-                <h3>Global Repositories:</h3>
-                <p>Global Package Repositories are available for all Kubeapps users.</p>
-                {globalRepos.length ? (
-                  <Table
-                    valign="center"
-                    columns={tableColumns}
-                    data={getTableData(globalRepos, !canEditGlobalRepos)}
-                  />
-                ) : (
-                  <p>
-                    There are no <i>global</i> Package Repositories yet. Click on the "Add Package
-                    Repository" button to create one.
-                  </p>
-                )}
-                {namespace !== globalReposNamespace && (
-                  <>
-                    <h3>Namespaced Repositories: {namespace}</h3>
+      <div className="catalog-container">
```

Comment:
Fixing a css issue: leaving more space at the bottom.

![image](https://user-images.githubusercontent.com/11535726/174052454-f2d98796-6a31-4f60-b477-42dd418edf87.png)

Reply:
Great, thanks.